sdks/python/apache_beam/utils/subprocess_server.py

```diff
@@ -147,6 +157,13 @@ def stop_process(self):
       if self._process.poll() is None:
         self._process.kill()
       self._process = None
+      if self._stdout_file_name and os.path.exists(self._stdout_file_name):
+        try:
+          os.unlink(self._stdout_file_name)
+        except Exception as e:
+          logging.error((
+              'Could not remove temporary file %s due to %r' %
+              (self._stdout_file_name, e)))
```

Comment:
I think we should be able to omit this:

> It will be destroyed as soon as it is closed (including an implicit close when the object is garbage collected)


Or we can explicitly request it with `self._stdout_file.close()`

Reply:
I think we have to manually delete since we set "delete=False" when creating the temp file.